core-integration-test/src/test/groovy/org/openstack4j/api/AbstractSpec.groovy

```diff
@@ -18,7 +18,7 @@ import co.freeside.betamax.tape.yaml.TapePropertyUtils
 abstract class AbstractSpec extends Specification {
 
     def static String MODULEROOT = Paths.get("").toAbsolutePath().getParent().toString()
-    def static String TAPEROOT = "/" + MODULEROOT + "src/test/resources/tapes/"
+    def static String TAPEROOT = MODULEROOT + "/" + "src/test/resources/tapes/"
```

Comment:
That is strange, the absolute path should start with '/' so additional 
should be ignored and it will blow up on Windows completely. Can we 
agree on `MODULEROOT + "/src/test/resources/tapes/"`?


On 06/02/2016 02:53 PM, Arno Uhlig wrote:
>
> In 
> core-integration-test/src/test/groovy/org/openstack4j/api/AbstractSpec.groovy 
> <https://github.com/ContainX/openstack4j/pull/704#discussion_r65533145>:
>
> > @@ -18,7 +18,7 @@ import co.freeside.betamax.tape.yaml.TapePropertyUtils
> >  abstract class AbstractSpec extends Specification {
> >
> >      def static String MODULEROOT = Paths.get("").toAbsolutePath().getParent().toString()
> > -    def static String TAPEROOT = "/" + MODULEROOT + "src/test/resources/tapes/"
> > +    def static String TAPEROOT = MODULEROOT + "/" + "src/test/resources/tapes/"
>
> @olivergondza <https://github.com/olivergondza> I think it should be 
> TAPEROOT = "/" + MODULEROOT + "/src/test/resources/tapes/"
>
> With JDK 1.7 that reduces the amount of errors in the integration-test 
> module to 3 NPE's caused by UnknownHostExceptions. Very strange. I'm 
> currently looking into it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ContainX/openstack4j/pull/704/files/f5fa1dc540f634c819f3ff030be09fe6df3bea7a#r65533145>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AAMn-cJv-jnn3Dinr4_3njNQHIoAd0Woks5qHtJfgaJpZM4Ir3Cq>.
>